### ISSUE TITLE
f4map

### ISSUE THREAD

**Issue by KillerKingSolo** [organicmaps/organicmaps#8657]
![image](https://github.com/organicmaps/organicmaps/assets/71156279/33e3a43e-d209-4c11-9392-46c5bb41173c)
Would you guys ever partner with f4map? I believe that it would significantly improve the quality of maps and make them accurate. I still appreciate the style you have with your dark and light modes. However, I believe it would be interesting to see the integration with f4map.

**Comment by Jean-BaptisteC** [comment]
No we don't have partnership with f4map.
I'm not sure an f4map give an API for the 3D render.
And If we render everything in 3D, battery consumption can increase a lot.

**Comment by KillerKingSolo** [comment]
I think you could make it a toggle option and in that you could add filters with what you see best fit. Also you don't have the render distance so far thus not consuming battery.

**Comment by biodranik** [comment]
Organic Maps uses its own graphics rendering engine called Drape. Integrating any other engine requires complete code rewriting, and may take years of man-hours to finish.

What are your use-cases for Organic Maps where you need something from f4map? Can you please describe them in detail? How useful are these cases for other users?

**Comment by KillerKingSolo** [comment]
I would like to see the trees and more realistic building shape roofs

**Comment by Jean-BaptisteC** [comment]
See roofs and treee is it very useful during navigation?

**Comment by KillerKingSolo** [comment]
Navigation wise no but it builds a more accurate app and for trees it helps me find hiking spots.

**Comment by Jean-BaptisteC** [comment]
IMO contour lines layer is better to find hiking spots

**Comment by KillerKingSolo** [comment]
Yes but it not available in all countries such as Thailand. I understand if you don't want to add f4map compatibility because of its difficulty to change the code.

**Comment by Jean-BaptisteC** [comment]
Maybe can add contour lines in Thailand :)
CC @vng

**Comment by RedAuburn** [comment]
it's likely that we have no data source for Thailand yet, a DTM dataset is needed to generate contour lines
